.github/workflows/claude-scheduled-autofix.yml

```
@@ -0,0 +1,388 @@
+      group: claude-autofix-pr-${{ matrix.pr.number }}
+      cancel-in-progress: false
+    permissions:
+      contents: write
+      pull-requests: write
+      issues: write
+      # OIDC + CI results access for the Claude Code action.
+      id-token: write
+      actions: read
+    steps:
+      - name: ⬇️ Checkout the PR branch
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)